app/controllers/payments_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PaymentsController < ApplicationController
+  def index
+    @category = Category.find(params[:category_id])
+    @payments = @category.payments.order(created_at: :desc)
+  end
+
+  def show; end
```

Comment:
[REQUIRED] Kindly remove this since it is not in use in order to make your project more professional